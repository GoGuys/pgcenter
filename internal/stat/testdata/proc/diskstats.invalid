   7       0 loop0 95 0 2452 32 0 0 0 0 0 48
   7       1 loop1 4240 0 9320 1752 0 0 0 0 0 548
   8       0 sda 364890 90257 15905820 98256 5112729 4404633 312756632 3722346 0 3986076
   8      16 sdb 307367 16786 12135182 8494946 84857424 8238986 1047928880 1649683162 0 114626504
