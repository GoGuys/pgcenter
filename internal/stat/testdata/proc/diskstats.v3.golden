   7       0 loop0 95 0 2452 32 0 0 0 0 0 48 4 0 0 0 0 0 0
   7       1 loop1 4240 0 9320 1752 0 0 0 0 0 548 552 0 0 0 0 0 0
   8       0 sda 365208 90272 15920380 98361 5144476 4431757 316780704 3758592 0 4010804 1967660 390788 0 242426032 1702032 1452785 1458752145
   8      16 sdb 309388 16786 12151846 8569789 85675301 8382129 1058502704 1669180952 0 115656192 1572052888 0 0 0 0 4587 12547854
